Toolkit.py

```
@@ -1,321 +1,321 @@
+		logger.debug("Check \"mscorlib.dll\" is not exists...");
+		if (not os.path.isfile("./mscorlib.dll") or (Toolkit.GetHash('./mscorlib.dll') != cls.__resourceFileInfo["CoreLib"])):
+			info:dict[str,any] = cls.__resourceFileInfo["CoreLib"];
+			Toolkit.ResourceReleases(info["offset"], info["lenth"], "./mscorlib.dll");
+		# 检查字体文件是否存在
+		logger.debug("Check \"musync_data\\LXGW.ttf\" is not exists...");
+		if not os.path.isfile('./musync_data/LXGW.ttf' or (Toolkit.GetHash('./musync_data/LXGW.ttf') != cls.__resourceFileInfo["Font"])):
+			info:dict[str,any] = cls.__resourceFileInfo["Font"];
+			Toolkit.ResourceReleases(info["offset"], info["lenth"], './musync_data/LXGW.ttf');
+		# 检查字体是否安装
+		logger.debug("Check \"霞鹜文楷等宽\" is not installed...");
+		if not '霞鹜文楷等宽' in fonts:
```

Ruff (0.14.8)

Test for membership should be `not in`

Convert to `not in`

(E713)